UT-Georeferencing-Earthworks-12d-2

LeeGregory12d opened the pull request:
## Unit test

Cut and fill elements for bulk earthworks for a small part of a job and only semantics for the large voided object.

## Covered topics

*tick one/many*

- [x] project setup
- [x] georeferencing
- [ ] alignment semantics
- [ ] alignment geometry
- [ ] linear placement
- [ ] project breakdown structure / spatial structure
- [ ] terrain
- [x] triangulated irregular network
- [ ] open cross profile
- [ ] sectioned surface & solid
- [ ] pavement & course
- [ ] surface feature
- [x] earthworks
- [ ] geotechnics
- [ ] longitudinal products (barriers, guardrail, ...)
- [ ] interchange
- [ ] structural
- [ ] drainage
- [ ] signage
- [ ] road furniture
- [ ] rail furniture
- [ ] ports & waterways furniture
- [ ] bridge furniture

LeeGregory12d committed:
UT-Georeferencing-Earthworks-12d-1

cut and fill elements for bulk earthworks for a small part of a job and only semantics for the voided object..

LeeGregory12d commented:
I'm not certain how I upset my full requests - this #50 is for UT-Georeferencing-Earthworks-12d-2

LeeGregory12d committed:
Remove IfcEarthworksCut from Spatial Structure; update readme

pjanck commented:
Following the changes to the schema, (we are now at `RC3`), we ask you to update your IFC files accordingly. The schema can be found [here](https://github.com/bSI-InfraRoom/IFC-Specification/blob/main/IFC4x3_RC3.exp). The checking procedure for IFC files has been updated - please merge in the current state of the `main` branch.

LeeGregory12d committed:
Update UT-Georeferencing-Earthworks-12d-2 for RC3

LeeGregory12d committed:
Georeferencing-earthworks-2: remove "UT-" from files

AlexBrad1eyCT reviewed (changes requested):
It is Fine to have the IfcSolidStratum without geometry as it would just be unwanted noise within the file, but it is currently orphaned within the structure.

- [ ] `IfcRelContainedInSpatialStructure` the `IfcSolidStratum` under the `IfcSite` so it is not orphaned

LeeGregory12d committed:
Wrong name of IFC file in Georeferencing-Earthworks-12d-2

The name of the IFC didn't match that given in the readme.md.
So change the name of the IFC file.
